Wrappers/Python/cil/processors/PaganinProcessor.py

```diff
@@ -204,101 +204,68 @@ def check_input(self, data):
         
         if data.dtype!=np.float32:
             raise TypeError('Processor only support dtype=float32')
+        
+        cil_order = tuple(AcquisitionDimension.get_order_for_engine('cil',data.geometry))
+        if data.dimension_labels != cil_order:
+            raise ValueError("This processor requires CIL data order, consider using `data.reorder('cil')`")
```

Comment:
Looks like you have changed from this being a reccommendation to a requirement. Shall we  add the change to the changelog?